canvas_modules/common-canvas/src/common-properties/controls/toggle/toggle.scss

```diff
@@ -14,10 +14,13 @@
  * limitations under the License.
  */
 
+.bx--toggle__switch {
```

Comment:
This won't be correct once the above is fixed.  I think all the styling will need to be revert and 1 styling added to override Controlitem spacing.